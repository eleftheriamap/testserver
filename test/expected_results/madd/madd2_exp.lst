
test/exp/test/gen/madd/madd2_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	9139296b 	add	x11, x11, #0xe4a
   4:	9163a6f7 	add	x23, x23, #0x8e9, lsl #12
   8:	517e8421 	sub	w1, w1, #0xfa1, lsl #12
   c:	9b170567 	madd	x7, x11, x23, x1
  10:	8a000000 	and	x0, x0, x0
