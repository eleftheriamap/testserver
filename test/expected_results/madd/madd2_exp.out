Registers:
X00    = 0000000000000000
X01    = 00000000ff05f000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 00000008f41b9000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000e4a
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 00000000008e9000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x9139296b
0x00000004 : 0x9163a6f7
0x00000008 : 0x517e8421
0x0000000c : 0x9b170567
0x00000010 : 0x8a000000
